client/components/MusicControl.tsx

```diff
@@ -24,6 +34,29 @@ const MusicControl: React.FC = () => {
         }
     };
 
+    const handleFlagSong = (songID: number, favorite: boolean) => {
+        flagSong(songID, favorite, props.authKey)
+            .then(() => {
+                musicContext.toggleFlag();
+                if (favorite) {
+                    return toast.success('Song added to favorites');
+                }
+                toast.success('Song removed from favorites');
+            })
+            .catch((err) => {
+                if (favorite) {
+                    toast.error(
+                        '😞 Something went wrong adding song to favorites.'
+                    );
+                } else {
```

Comment:
The `else` could be avoided by using a `return`

Reply:
I copied this fragment from other components such as `SongList`

https://github.com/ampache/ampache/blob/f4749d26a3b24daaadc1e23f4d49ada2731ad7ef/client/components/SongList/index.tsx#L125-L129

Feel free to change my PR as you wish